docs/user/AzureHelpTopics.md

```diff
@@ -91,7 +98,7 @@ az identity create --resource-group <my resource group> --name <my identity name
 PID=$(az identity show --resource-group <my resource group> --name <my identity name> --output tsv --query principalId)
 az role assignment create --assignee $PID --role Contributor
 ```
-## Assign the Managed Identity to a VM
+## How to assign User-assigned Managed Identity to a VM
```

Comment:
add article: How to assign  _a_ User-assigned...

Reply:
can you elaborate?

Reply:
add the word "a" between "assign" and "User-assigned" (I think the grammatical term for that is "indefinite article" :) )